entc/integration/cascadelete/ent/comment_query.go

```diff
@@ -327,6 +328,7 @@ func (cq *CommentQuery) GroupBy(field string, fields ...string) *CommentGroupBy
 //	client.Comment.Query().
 //		Select(comment.FieldText).
 //		Scan(ctx, &v)
+//
```

Comment:
Now, that I see this, can you please go generate with go version 1.18? I'd like to have the step for 1.19 done in a separate PR.